tests/playwright-test/playwright.artifacts.spec.ts

```diff
@@ -189,6 +189,71 @@ test('should work with screenshot: only-on-failure', async ({ runInlineTest }, t
   ]);
 });
 
+test('should work with screenshot: on & fullPage & omitBackground', async ({ runInlineTest }, testInfo) => {
```

Comment:
Instead of these tests, I'd rather test that passing `fullPage` does indeed take a full page screenshot.

I'd recommend to combine the page setup of [this test](https://github.com/microsoft/playwright/blob/main/tests/page/page-screenshot.spec.ts#L214-L219) and `expect.toMatchSnapshot` the generated screenshot in the output folder. You can find the generated snapshot at something like `testInfo.outputPath('test-results', 'filename-testname', 'test-failed-1.png')`.

Reply:
Makes sense, thanks! Just updated the test.